mirobo/strip.py

```diff
@@ -0,0 +1,62 @@
+from .device import Device
+from typing import Any, Dict
+
+
+class Strip(Device):
+    """Main class representing the smart strip."""
+
+    def on(self):
+        """Power on."""
+        return self.send("set_power", ["on"])
+
+    def off(self):
+        """Power off."""
+        return self.send("set_power", ["off"])
+
+    def status(self):
+        """Retrieve properties."""
+        properties = ['power', 'temperature', 'current', 'mode']
+        values = self.send(
+            "get_prop",
+            properties
+        )
+        return StripStatus(dict(zip(properties, values)))
+
+    def set_power_mode(self, mode: str):
+        """Set mode."""
```

Comment:
Btw, you can then simply use mode.value to get the value on the right hand side insie `set_mode()`, and other way around when reading it, PowerMode('green') will give you back the PowerMode.Eco :-)